airflow/providers/amazon/aws/secrets/systems_manager.py

```diff
@@ -100,24 +98,6 @@ def get_conn_value(self, conn_id: str) -> Optional[str]:
 
         return self._get_secret(self.connections_prefix, conn_id)
 
-    def get_conn_uri(self, conn_id: str) -> Optional[str]:
-        """
-        Return URI representation of Connection conn_id.
-
-        As of Airflow version 2.3.0 this method is deprecated.
-
-        :param conn_id: the connection id
-        :return: deserialized Connection
-        """
-        if get_airflow_version() >= (2, 3):
-            warnings.warn(
-                f"Method `{self.__class__.__name__}.get_conn_uri` is deprecated and will be removed "
-                "in a future release.  Please use method `get_conn_value` instead.",
-                DeprecationWarning,
-                stacklevel=2,
-            )
-        return self.get_conn_value(conn_id)
-
```

Comment:
This is the reason why SSM Parameters Sectet Backend Task failed.
I don't think we can remove this method right now, otherwise it would use `BaseSecretsBackend.get_conn_uri` which raise NotImplementedError

Also Provider might use in Airflow 2.2 where this method not deprecated

Reply:
Keeping it